### ISSUE TITLE
Can not specify range from any weekday to sunday

### ISSUE THREAD

**Issue by egfx-notifications** [Hexagon/croner#82]
A pattern like `* * * * FRI-SUN` will not be valid because `SUN` is replaced with `0` and thus the following error is thrown
```
TypeError: CronPattern: From value is larger than to value: '5-0'
```

Since alphabetical weekdays are non-standard syntax anyway, would you be willing to have a fix for this edge case added?

A likely candidate could be here
https://github.com/Hexagon/croner/blob/9f23fae0304d02ab346b142504a9b1a7a49a5e78/src/pattern.js#L315-L324

I guess something like this should work, but did not run all tests yet
```
CronPattern.prototype.replaceAlphaDays = function (conf) {
	return conf
		.replace(/-sun/gi, "-7")
		.replace(/sun/gi, "0")
		.replace(/mon/gi, "1")
		.replace(/tue/gi, "2")
		.replace(/wed/gi, "3")
		.replace(/thu/gi, "4")
		.replace(/fri/gi, "5")
		.replace(/sat/gi, "6");
};
```

This way we keep `0` as the standard for sundays, but automatically choose `7` for upper values of ranges.

The following would be an even cleaner alternative, but lookahead and lookbehind in regex are sometimes unsupported in older systems, so depending on what should still be supported that is not an option
```
CronPattern.prototype.replaceAlphaDays = function (conf) {
	return conf
		.replace(/(?<=-)sun/gi, "7")
		.replace(/sun/gi, "0")
		.replace(/mon/gi, "1")
		.replace(/tue/gi, "2")
		.replace(/wed/gi, "3")
		.replace(/thu/gi, "4")
		.replace(/fri/gi, "5")
		.replace(/sat/gi, "6");
};
```

**Comment by Hexagon** [comment]
Interesting finding! Submit a PR and we'll take it from there 👍

**Comment by egfx-notifications** [comment]
On it.

The regex with lookbehind works for `npm run test`, but eslint complains about it in `npm run build`, so I'll stick with the backwards-compatible version.

**Comment by Hexagon** [comment]
@egfx-notifications 

Released in [4.3.9](https://github.com/Hexagon/croner/releases/tag/4.3.9) 👍
